Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs

```diff
@@ -1,36 +1,48 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
+﻿using System.Linq;
 using Xamarin.Forms.CustomAttributes;
 using Xamarin.Forms.Internals;
+using System;
+using System.Threading.Tasks;
+
+#if UITEST
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+#endif
 
 namespace Xamarin.Forms.Controls.Issues
 {
+#if UITEST
+	[Category(UITestCategories.Layout)]
+#endif
 	[Preserve (AllMembers=true)]
 	[Issue (IssueTracker.Github, 1758, "LayoutTo needs to be smarted about using layout specific API calls", PlatformAffected.Android | PlatformAffected.iOS | PlatformAffected.WinPhone)]
-	public class Issue1758 : ContentPage
+	public class Issue1758 : TestContentPage
 	{
 		ListView _list;
 		Button _button;
- 
-		public Issue1758()
+
+		protected override void Init()
 		{
 			_list = new ListView { ItemsSource = new[] { "hello", "world", "from", "xamarin", "forms" } };
- 
-			_button = new Button { Text = "Button" };
- 
+
+			_button = new Button { Text = "Button", AutomationId = "Button" };
```

Comment:
To prevent magic string mistakes could you make the automation id a private const and use that?